subxt/src/utils/account_id.rs

```diff
@@ -37,9 +40,9 @@ impl From<[u8; 32]> for AccountId32 {
 }
 
 impl AccountId32 {
-    // Return the ss58-check string for this key. Adapted from `sp_core::crypto`. We need this to
-    // serialize our account appropriately but otherwise don't care.
-    fn to_ss58check(&self) -> String {
+    /// Return the ss58-check string for this key. Adapted from `sp_core::crypto`. We need this to
+    /// serialize our account appropriately but otherwise don't care.
+    pub fn to_ss58check(&self) -> String {
```

Comment:
This and `from_ss58_check` are already exposed via `.to_string()` and `.parse()`.

One reason I'm a little hesitant to expose the actual methods is because you can't set the chain prefix; it's always "Substrate" (which is fine for display/FromStr and fine for serializing them to get the account nonce, since it doesn't matter there, but not so good as a public API).

Reply:
(that all said, perhaps it is worth copying across the "full" to_ss58 function so that it's possible to do this; in this original cut I was just adding the bare minimum needed to make things work, really..)

Reply:
I will remove this here and keep the serialize/deserialize which is most critical for my use case.